src/game.js

```diff
@@ -1,46 +1,44 @@
-/*	Game Class
+/* Game Class
  *
- *	Game contains all Game elements and Game mechanisms functions.
- *	It's the root element and defined only one time through the G variable.
+ * Game contains all Game elements and functions.
+ * It's the root element and defined only one time through the G variable.
  *
- *	NOTE: Constructor does nothing because the G object must be defined
- *	before creating other classes instances. The game setup is triggered
- *	to really start the game.
+ * NOTE: Constructor does nothing because the G object must be defined
+ * before creating other class instances. The game setup is triggered
+ * to really start the game.
  *
  */
 
 var Game = Class.create({
-	/*	Attributes
+	/* Attributes
 	 *
-	 *	NOTE : attributes and variables starting with $ are jQuery elements
-	 *	and jQuery functions can be called directly from them.
+	 * NOTE : attributes and variables starting with $ are jQuery elements
+	 * and jQuery functions can be called directly from them.
 	 *
-	 *	// jQuery attributes
-	 *	$combatFrame :	Combat element containing all graphics except the UI
+	 * // jQuery attributes
+	 * $combatFrame : Combat element containing all graphics except the UI
 	 *
-	 *	// Game elements
-	 *	players :			Array :	Contains Player objects ordered by player ID (0 to 3)
-	 *	creatures :			Array :	Contains Creature objects (creatures[creature.id]) start at index 1
+	 * // Game elements
+	 * players : Array : Contains Player objects ordered by player ID (0 to 3)
+	 * creatures : Array : Contains Creature objects (creatures[creature.id]) start at index 1
 	 *
-	 *	grid :				Grid :	Grid object
-	 *	UI :				UI :	UI object
+	 * grid : Grid : Grid object
+	 * UI : UI : UI object
 	 *
-	 *	queue :				CreatureQueue :	queue of creatures to manage phase order
+	 * queue : CreatureQueue : queue of creatures to manage phase order
 	 *
-	 *	turn :				Integer :	Current's turn number
-	 *
-	 *	// Normal attributes
-	 *	playerMode :		Integer :	Number of players in the game
-	 *	activeCreature :	Creature :	Current active creature object reference
-	 *	creaIdCounter :		Integer :	Creature ID counter used for creature creation
-	 *	creatureData :		Array :		Array containing all data for the creatures
+	 * turn : Integer : Current's turn number
 	 *
+	 * // Normal attributes
+	 * playerMode : Integer : Number of players in the game
+	 * activeCreature : Creature : Current active creature object reference
+	 * creaIdCounter : Integer : Creature ID counter used for creature creation
+	 * creatureData : Array : Array containing all data for the creatures
 	 */
```

Comment:
This part uses tabs to nicely arrange stuff almost like in a table fashion so that's easier visible.
![screenshot from 2017-10-07 07-55-32](https://user-images.githubusercontent.com/587487/31304864-0086efc4-ab35-11e7-9f6f-3dfd5ffea22f.png)
Was it the wrong thing to do? :)